lib/fog/vsphere/requests/compute/create_vm.rb

```diff
@@ -7,6 +7,7 @@ def create_vm attributes = { }
 
           vm_cfg        = {
             :name         => attributes[:name],
+            :annotation   => attributes[:annotation],
```

Comment:
Good idea! I've done this and tested it works OK.

(I was only thinking about creating VMs, not querying them.)